### DISCUSSION TITLE
Trying to upgrade the Tailwindcss in Nextjs v14 Project

### DISCUSSION THREAD

**Discussion by peyman-dev** [tailwindlabs/tailwindcss#15317] [Help]
I'm just trying to upgrade the tailwindcss in nextjs project, the styles changes successfully But when im trying to save and compile the code, I'm facing with this error in terminal:

`Import trace for requested module:
./app/globals.css.webpack[javascript/auto]!=!./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[14].oneOf[12].use[2]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[14].oneOf[12].use[3]!./app/globals.css`

![image](https://github.com/user-attachments/assets/c67cebad-2c61-489d-aea0-31eb8635d667)

**Comment by philipp-spiess** [comment]
Hey! Mind sharing a reproduction setup that we can look into? :)

**Reply by peyman-dev** [reply]
Thanks for your response! If possible, please check out this [project](https://github.com/peyman-dev/skycode) 🙏

Thanks again for your help!

**Reply by philipp-spiess** [reply]
Thanks! Two things:

- This project is already migrated to tailwind 4 it seems
- It works just fine on my machine 😬 

What am I missing? 

<img width="865" alt="Screenshot 2024-12-06 at 12 23 02" src="https://github.com/user-attachments/assets/321e422b-ddfd-4e46-85fa-1fe05e0cc84a">

**Reply by peyman-dev** [reply]
Can you please check out the terminal? 
After any "Ctrl + S" im facing with this message/error in terminal:

![image](https://github.com/user-attachments/assets/645f1110-f67e-4c0f-a1a0-9203119d3f2c)

**Reply by philipp-spiess** [reply]
I don't see any such output in the CLI I'm afraid :/ 
<img width="699" alt="Screenshot 2024-12-06 at 12 59 04" src="https://github.com/user-attachments/assets/55b30481-e6b1-45a9-906d-2cb0ab58f7f6"> I started the server via `npm run dev`. Let me test this on Windows too, since the file paths in your example might suggest you're running this on Windows!

**Reply by philipp-spiess** [reply]
Ah yes this reproduces on Windows! We'll look into it. Thanks for reporting this issue!

**Reply by peyman-dev** [reply]
Well, thank you for your time and help. I did my job, and I hope this problem gets fixed as soon as possible.💚

**Reply by philipp-spiess** [reply]
Already testing the fix #15321, thank you 🙇‍♂️

**Reply by peyman-dev** [reply]
Philip,
Sorry to bother you, but I have a question.
Do I need to add something to the project manually, or has this issue been resolved globally?
I’m still encountering this error. 
### ERROR 👇
![image](https://github.com/user-attachments/assets/54b04dff-919b-49ca-96b5-c05d329eec92)

**Reply by philipp-spiess** [reply]
@peyman-dev Hey! This fix was just merged into the code and will be part of the next release. No exact ETA yet but we're hoping to get another round of fixes out later this week.
